src/mpid/ch4/netmod/ofi/ofi_am_events.h

```diff
@@ -152,13 +152,31 @@ static inline int MPIDI_OFI_do_rdma_read(void *dst,
         comm = MPIDI_CH4U_context_id_to_comm(context_id);
         MPIR_Assert(comm);
         MPIDI_OFI_cntr_incr();
-        MPIDI_OFI_CALL_RETRY_AM(fi_read(MPIDI_OFI_EP_TX_RMA(0),
-                                        (char *) dst + done,
-                                        curr_len, NULL,
-                                        MPIDI_OFI_comm_to_phys(comm, src_rank, MPIDI_OFI_API_TAG),
-                                        src + done,
-                                        MPIDI_OFI_AMREQUEST_HDR(rreq, lmt_info).rma_key,
-                                        &am_req->context), FALSE /* no lock */ , read);
+
+        struct iovec iov = {
```

Comment:
It's C99 standard, and it is supported by most of Unix/Linux compilers released after 2012.
such initialization is one of the best innovation in C99 :)